libr/core/cmd_debug.c

```diff
@@ -2828,7 +3002,7 @@ static int cmd_debug(void *data, const char *input) {
 		char str[128];
 		str[0] = 0;
 		r_print_date_get_now (core->print, str);
-		r_cons_println (str);
+		r_cons_printf ("%s\n", str);
```

Comment:
don't remove r_cons_println in favor of r_cons_printf. Here the commit and its reason https://github.com/radare/radare2/commit/c1457400308a98f88bed2a4b8f5b533f12dc8a77